kit/src/main/java/com/oracle/javafx/scenebuilder/kit/metadata/Metadata.java

```diff
@@ -4470,7 +4472,12 @@ public boolean isPropertyTrimmingNeeded(PropertyName name) {
                 com.oracle.javafx.scenebuilder.kit.metadata.property.value.DoublePropertyMetadata.DoubleKind.COORDINATE,
                 true, /* readWrite */
                 100.0, /* defaultValue */
-                new InspectorPath("Properties", "Specific", 98));
+                new InspectorPath("Properties", "Specific", 10));
+    private final ValuePropertyMetadata useSystemMenuBarPropertyMetadata =
```

Comment:
Have you tested this on Windows or Linux? The property is public API, so it shouldn't have any effect.

Reply:
So, the test on Windows 10 went fine, the use of system menu bar can be toggled, it saves correctly. When disabled, the property is not set to false, it is removed, which is okay. There is no visual effect in preview, regardless of having the property is enabled or not.

It works for Windows.

Reply:
Great!

Reply:
Just finished a test on Ubuntu 20.04 and also here, it works fine.